webhooks/credit_invoices.go

```diff
@@ -24,18 +24,26 @@ type CreditInvoiceNotification struct {
 	Invoice CreditInvoice `xml:"invoice"`
 }
 
+// https://dev.recurly.com/page/webhooks#section-credit-invoice-schema
```

Comment:
Comment is really helpful but needs to be moved down one line -- first line of the godoc comment should be the `CreditInvoice` line

Reply:
done